mail/src/main/java/jakarta/mail/internet/MimeMessage.java

```diff
@@ -1716,7 +1716,6 @@ public Message reply(boolean replyToAll, boolean setAnswered)
 		eliminateDuplicates(v,
 				InternetAddress.parse(alternates, false));
 	    // should we Cc all other original recipients?
-	    String replyallccStr = null;
```

Comment:
Became unused in https://github.com/javaee/javamail/commit/d9b77024e98795a753e3d95f29164112579d5e3a